fulp_modules/features/toys/code/plushies.dm

```diff
@@ -87,3 +87,25 @@
 	icon = 'fulp_modules/features/toys/icons/toys.dmi'
 	icon_state = "beefman"
 	squeak_override = list('sound/effects/meatslap.ogg'=1)
+
+//Do your cooldown changes here.
+#define BEEFPLUSHIE_COOLDOWN_TIME (2 MINUTES)
+
+/obj/item/toy/plush/beefplushie/living_beefplushie
+	desc = "It looks oddly alive. You feel like you should pet it."
+	COOLDOWN_DECLARE(beefplushie_cooldown)
+
+//When used in hand.
+/obj/item/toy/plush/beefplushie/living_beefplushie/attack_self(mob/user)
+	. = ..()
+	if(!COOLDOWN_FINISHED(src, beefplushie_cooldown))
+		user.balloon_alert(user, "not ready yet!")
+		return
+	user.balloon_alert(user, "producing meat")
+	if(!do_after(user, 3 SECONDS, target = src))
+		return
+	playsound(src, "sound/effects/splat.ogg", 50)
+	new /obj/item/food/meat/slab get_turf(loc)
```

Comment:
![qFZ](https://user-images.githubusercontent.com/68669754/150873935-e613f4e6-e87e-4190-93cc-67f7c56c0763.gif)

where's the parenthesis?